e2e/interchaintestv8/e2esuite/light_clients.go

```diff
@@ -0,0 +1,348 @@
+package e2esuite
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+
+	ibcwasmtypes "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	mock "github.com/cosmos/ibc-go/v8/modules/light-clients/00-mock"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/testutil"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/ethereum"
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
+	ethereumligthclient "github.com/srdtrk/solidity-ibc-eureka/e2e/v8/types/ethereumlightclient"
```

Comment:
nit: ethereumligthclient should be ethereumlightclient

Reply:
:O Will def get this in the next PR 🙏